docs/docs/grpc.md

```diff
@@ -18,12 +18,134 @@ main = runGRpcApp 8080 "helloworld" quickstartServer
 
 ## Building a client
 
--- TODO:
+Right now there are two options for building clients: using records or with `XTypeApplications`. To give a proper introduction to both options let's consider in detail an example client for the following services:
```

Comment:
Usually, one writes either `TypeApplications` (because that's the parameter to `LANGUAGE` pragmas) or `-XTypeApplications` (because that's the one you enable it in ghci).